src/PartialModelSelection.hpp

```diff
@@ -27,34 +27,32 @@ struct MinimizeResult {
     std::pair<double, double> optimalModels;
 };
 
+    //Map constants and return codes.
+    const double PLACEHOLDER_LOSS = -9999.0;
+    const double EMPTY_MAP_QUERY = 0;
+    const double EMPTY_MAP_ERR = -99999;
+    const double DEFAULT_PENALTY = -9999;   
+    const int STD_MODEL_CAP = 3;
```

Comment:
why do you need these? you should avoid arbitrary constants, please delete and refactor.

Reply:
Yeah most of them are no longer necessary, will do.